// LANGUAGE: +InlineClasses, -JvmInlineValueClasses, +SealedInlineClasses
// SKIP_TXT

sealed inline class IC

<!INLINE_CLASS_UNDERLYING_VALUE_IS_SUBCLASS_OF_ANOTHER_UNDERLYING_VALUE!>sealed<!> inline class ICC: IC()

<!INLINE_CLASS_UNDERLYING_VALUE_IS_SUBCLASS_OF_ANOTHER_UNDERLYING_VALUE!>sealed<!> inline class ICC2: IC()
